src/containers/SortingUnitView.tsx

```diff
@@ -218,7 +218,7 @@ const mapStateToProps: MapStateToProps<StateProps, OwnProps, RootState> = (state
     recording: findRecordingForId(state, (findSortingForId(state, ownProps.sortingId) || {}).recordingId),
     extensionsConfig: state.extensionsConfig,
     documentInfo: state.documentInfo,
-    plugins: state.plugins,
+    plugins: filterPlugins(state.plugins),
```

Comment:
Exclude plugins that have `disabled: true` or `development: true`. Later we will find a mechanism for enabling the development plugins.